From 45d483550c53b7eff76c0461cb292a1f4ab5c6b5 Mon Sep 17 00:00:00 2001
Message-Id: <45d483550c53b7eff76c0461cb292a1f4ab5c6b5.1684964119.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1684964119.git.sharlatanus@gmail.com>
References: <cover.1684964119.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 24 May 2023 22:31:56 +0100
Subject: [PATCH v2 20/20] gnu: python-asdf: Enable tests.

* gnu/packages/astronomy.scm (python-asdf): Enable the most tests.
---
 gnu/packages/astronomy.scm | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index e3ebaee8b4..67b5a601e3 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2738,12 +2738,13 @@ (define-public python-asdf
         (base32 "11s56797l5330kkhppkyz0bsvms016knmyswj4gx91zrxf8iqvv8"))))
     (build-system pyproject-build-system)
     (arguments
-     ;; FIXME: Tests fail a lot with
-     ;;
-     ;; ERROR  - _pytest.pathlib.ImportPathMismatchError:
-     ;; ('asdf.conftest', '/gnu/sto...
-     ;;
-     `(#:tests? #f))
+     (list
+      #:test-flags
+      #~(list "-k" (string-append
+                    "not test_overwrite"
+                    " and not test_tagging_scalars"
+                    " and not test_info_command"
+                    " and not test_array_inline_threshold_recursive"))))
     (native-inputs
      (list python-astropy
            python-fsspec
-- 
2.39.2

